contracts/Locking.sol

```diff
@@ -157,7 +185,7 @@ contract Locking is Ownable, ReentrancyGuard {
         return _totalBoosted / PRECISION;
     }
 
-    function totalBoostedAt(uint256 month) private view returns (uint256) {
+    function totalBoostedAt(uint256 month) public view returns (uint256) {
```

Comment:
we will need this for external contracts. also I wanted to add a test using this.